docs/yaml_configuration.rst

```diff
@@ -349,6 +349,10 @@ REST API
    -  **http\_extra\_headers**: (optional): HTTP headers let the REST API server pass additional information with an HTTP response.
    -  **https\_extra\_headers**: (optional): HTTPS headers let the REST API server pass additional information with an HTTP response when TLS is enabled. This will also pass additional information set in ``http_extra_headers``.
    -  **request_queue_size**: (optional): Sets request queue size for TCP socket used by Patroni REST API.  Once the queue is full, further requests get a "Connection denied" error. The default value is 5.
+   -  **server_tokens**: (optional): Configures the value of the ``Server`` HTTP header.
```

Comment:
You also need to update https://github.com/patroni/patroni/blob/master/docs/ENVIRONMENT.rst#rest-api